tetrateci/tsb-util/k8s-objects/role.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  namespace: {{ targetNS }}
+  name: service-reader
+rules:
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["get"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: service-reader-binding
+  namespace: {{ targetNS }}
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
create a "trafficgen" serviceaccount and use that account in the deployment. Don't use 'default'